### PR TITLE
Add docker support and automated build through github actions

### PR DESCRIPTION
**Pull request by yodapotatofly**
This allows to generate a scirius docker image on each push/PR/release/tag event, named after both the commit sha and the branch/release/tag name

### CODE REVIEW

**Comment by regit** [comment]
Can you prefix the commit message with 'github:' ?

File: docker/scirius/suricata/scripts/suri_reloader [hunk 1]
```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/python
```
Comment (regit) [inline docker/scirius/suricata/scripts/suri_reloader:1]: This is already part of scirius code so we don't need to duplicate.

File: docker/scirius/scirius/local_settings.py [hunk 2]
```diff
@@ -0,0 +1,168 @@
+# Copyright (C) 2019, Stamus Networks
```
Comment (regit) [inline docker/scirius/scirius/local_settings.py:1]: We need copyright of original author and the one from Stamus to be added.

File: docker/scirius/bin/start-scirius.sh [hunk 3]
```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
```
Comment (regit) [inline docker/scirius/bin/start-scirius.sh:1]: This is mostly some code from Stamus: https://github.com/StamusNetworks/scirius-docker/blob/master/django/scirius.sh so we can update copyright date.

File: Dockerfile [hunk 4]
```diff
@@ -0,0 +1,168 @@
+# Copyright(C) 2021, Stamus Networks and Gabor Seljan
```
Comment (regit) [inline Dockerfile:1]: Better to do on ocpyrhig:
```
Copyright(C) 2020, Gabor Seljan
Copyright(C) 2021, Stamus Networks
```
Simply guessing on date.

**Review by regit — CHANGES_REQUESTED** [review]
There is some issue with the various copyright and we need a prefix on commit messages.

File: docker/scirius/suricata/scripts/suri_reloader (hunk 1 above)
Comment (yodapotatofly) [reply docker/scirius/suricata/scripts/suri_reloader:1]: This version is quite different: I has been adapted to use the UNIX socket (according to the path in env variables) and different path have been adapted as well

**Comment by yodapotatofly** [comment]
> 
> 
> Can you prefix the commit message with 'github:' ?

Not with 'docker:' ?

**Comment by regit** [comment]
Hello @yodapotatofly 

Can you try to get this MR to a lower number of commits and remove the Merge ones ?

**Comment by yodapotatofly** [comment]
should I squash even more ?

**Comment by regit** [comment]
Merged, thanks a lot Raphael!

**Pull request closed without merging**